packages/amplify-cli/src/commands/upgrade.ts

```diff
@@ -78,17 +78,6 @@ const upgradeCli = async (print, version: string) : Promise<void> => {
   await downloadPromise;
   await fs.move(extractedPath, binPath, { overwrite: true });
   await fs.chmod(binPath, '700');
-  // Load the new version of amplify-cli into memory to optimize subsequent runs and check if it runs without error.
-  const spinner:ora.Ora = ora('Validating downloaded Amplify CLI....');
```

Comment:
If this is the only usage of ora, let's remove the import too

Reply:
Great catch,  removed ora and execa